.github/workflows/test-demo.yml

```diff
@@ -0,0 +1,19 @@
+name: test-demo
```

Comment:
should we move this to `test.yaml` instead of having two separate test workflows?